.env.example

```diff
@@ -1,4 +1,4 @@
 VTOOLS_API="https://events.vtools.ieee.org/RST/events/api/public/<vtools_event_api_version>/"
 CALENDAR_ID="<google_calendar_collab_email>"
-COUNTRY_ID=<vtools_events_country_id>
-FILTER_MODE="COUNTRY"
+COUNTRY_CODE="<ISO_3166-1_alpha-2_code>"

```

Comment:
This implementation is making me thing we should instead look for arguments instead of environment variables. I.e., running the script with `-c <country_id>` or `--country-code <iso-code>`.